### ISSUE TITLE
allow setting multiple controllers

### ISSUE THREAD

**Issue by maxadamo** [eduvpn/vpn-server-node#53]
I wonder if having multiple controllers is feasible at all and if it's on your road map. 
It happens quite a few times, that maintenance works in the controllers, brings down both VPNs. 

In our two-nodes setup, it would be great if both nodes had the ability to be a controller. 
I understand that the controller is not a stateless application. For instance, the SQL Lite DBs are stored locally in the controller.

**Comment by fkooman** [comment]
Yes, it will be part of 3.x, it is not fully documented yet, but getting there!

Start of documentation: https://github.com/eduvpn/documentation/blob/v3/HA.md

**Comment by maxadamo** [comment]
thanks @fkooman . 
Feel free to close the issue.
